daisy_workflows/windows_upgrade/upgrade_script.ps1

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
Will this script work for other Windows upgrade paths (e.g. 2012->2016)? If not, the name of this script should reflect the specific upgrade path it's implementing.